src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestHearingRequirementsDirectionHandler.java

```diff
@@ -90,4 +91,11 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
 
         return new PreSubmitCallbackResponse<>(asylumCase);
     }
+
+    private Parties resolvePartiesForHearingRequirements(AsylumCase asylumCase) {
+        if (HandlerUtils.isAipJourney(asylumCase) || isInternalCase(asylumCase) && isAppellantInDetention(asylumCase) && !isAcceleratedDetainedAppeal(asylumCase)) {
```

Comment:
(HandlerUtils.isAipJourney(asylumCase) || 
(isInternalCase(asylumCase) && isAppellantInDetention(asylumCase) && !isAcceleratedDetainedAppeal(asylumCase))) 
Can we please change the condition.